server/repository/src/migrations/v1_06_00/master_list_line_add_item_link_id.rs

```diff
@@ -0,0 +1,41 @@
+use crate::{migrations::sql, StorageConnection};
+
+pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    if cfg!(feature = "postgres") {
+        sql!(
+            connection,
+            r#"
+            ALTER TABLE master_list_line 
+            ADD COLUMN item_link_id TEXT NOT NULL DEFAULT 'temp for migration';
```

Comment:
pedantic but consistency with other parts of this feature's migrations heh 
```suggestion
            ADD COLUMN item_link_id TEXT NOT NULL DEFAULT 'temp_for_migration';
```